server/server.js

```diff
@@ -56,35 +60,38 @@ exports = {
     }
     issue_priority = priorityMap[`${priority_no}`];
```

Comment:
Unnecessary string interpolation again

```suggestion
   const issue_priority = priorityMap[priority_no] || 'trivial';
```